src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -478,6 +478,12 @@ private function createFirewall(ContainerBuilder $container, string $id, array $
                 ->replaceArgument(0, new Reference($managerId))
             ;
 
+            // user checker listener
+            $container
+                ->setDefinition('security.listener.user_checker.'.$id, new ChildDefinition('security.listener.user_checker'))
+                ->replaceArgument(0, new Reference($firewall['user_checker']))
```

Comment:
Should se inject the generated alias instead?
https://github.com/symfony/symfony/blob/72a192c84f14070846ecd992237d38b4d4b82b05/src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php#L509

Reply:
I don't see an alias being generated somewhere. What would the benefit be of using an alias here?

Reply:
I mean:
```suggestion
                ->replaceArgument(0, new Reference('security.user_checker.'.$id))
```

This alias is created at https://github.com/symfony/symfony/blob/5.1/src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php#L503

Reply:
Oh sorry, I mis-understood your previous message. PR updated